db_schema/schema.sql

```diff
@@ -156,11 +156,11 @@ DROP TABLE IF EXISTS appleEpisodeDetails;
 CREATE TABLE appleEpisodeDetails (
   account_id INTEGER NOT NULL,
   episode_id BIGINT NOT NULL,
-  aed_playscount INTEGER NOT NULL,
-  aed_totaltimelistened BIGINT NOT NULL,
-  aed_uniqueengagedlistenerscount INTEGER NOT NULL,
-  aed_uniquelistenerscount INTEGER NOT NULL,
-  aed_engagedplayscount INTEGER NOT NULL,
+  aed_playscount INTEGER,
```

Comment:
why did you remove NOT NULL? do you set it to NULL, it should be just 0, right?

Reply:
if we convince ajv to not only validate the schema but also edit the data, this should be obsolete (see other comment including the link to the option for ajv)

Reply:
true; added the option and reverted the schema changes. Tested and works.